### DISCUSSION TITLE
Disable all shortcuts that affect gameplay while playing

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#19758] [Ideas]
I have a friend who told me that he accidentally touches `Toggle skin editor` (`Ctrl+shift+s`) while playing mania

he said mostly mania player use zx ./ or as;' for 4k play

Although this problem can be solved by changing the shortcut keys, but I think we can disable all shortcut except pause, quick retry, volume adjust while gameplaying

and re-enable when break and pausing

**Comment by peppy** [comment]
How on earth do you hit ctrl+shift+s accidentally?

Skin editor is intentionally available in this case (note that other hotkeys *are* disabled) because it can be used to test or modify a skin.
